tests/html/tiny_test.html

```diff
@@ -1 +1 @@
-  
```

Comment:
I'm not sure; it was as such when the file was created via https://github.com/servo/servo/commit/c6ab60dbfc6da7b4f800c9e40893c8b58413960c. Will try to find out.